docs/rules/relative-font-units.md

```diff
@@ -0,0 +1,103 @@
+# relative-font-units
+
+Enforce the use of relative units for font size.
+
+## Background
+
+The `font-size` property in CSS defines the size of the text. It can be set using:
+
+1. Keywords (e.g., `small`, `medium`, `large`).
+1. Length units (e.g., `px`, `em`, `rem`, `pt`).
+1. Percentages (`%`, relative to the parent element's font size).
+
+Generally, relative unit such as `rem` or `em` are preferred over absolute ones like `px`, `pt` because of following reasons:
```

Comment:
```suggestion
Generally, relative units such as `rem` or `em` are preferred over absolute ones like `px`, `pt` because of the following reasons:
```